ROMFS/px4fmu_common/init.d/airframes/1103_standard_vtol_sih.hil

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+#
+# @name SIH Standard VTOL
+#
+# @type Simulation
+# @class VTOL
+#
+# @output Motor1 motor right
```

Comment:
Needs to be updated

Reply:
done [here](https://github.com/PX4/PX4-Autopilot/pull/24175/commits/516a0cc86ad8234a0e2b8d4dbf8db51558481c16). also in 1002.